Makefile

```diff
@@ -266,6 +268,11 @@ mockgen: $(MOCKGEN) ## Download mockgen locally if necessary
 $(MOCKGEN): $(LOCALBIN)
 	GOBIN=$(LOCALBIN) go install github.com/golang/mock/mockgen@v1.6.0
 
+.PHONY: applygingersnapstyle-gen
+applygingersnapstyle-gen: $(APPLYGINGERSNAPSTYLE_GEN) ## build applygingersnapstyle-gen	 locally if necessary
+$(APPLYGINGERSNAPSTYLE_GEN): $(LOCALBIN)
+	pushd gingersnap-api/cmd/applygingersnapstyle-gen && GOBIN=$(LOCALBIN) go install && popd
```

Comment:
Is there a need to use `push` pod here for a one-liner, or can we just do:

`cd gingersnap-api/cmd/applygingersnapstyle-gen; GOBIN=$(LOCALBIN) go install`